code/__HELPERS/icons.dm

```diff
@@ -1038,32 +1038,26 @@ GLOBAL_LIST_EMPTY(friendly_animal_types)
 /image/proc/setDir(newdir)
 	dir = newdir
 
-#define FROZEN_RED_COLOR "#2E5E69"
-#define FROZEN_GREEN_COLOR "#60A2A8"
-#define FROZEN_BLUE_COLOR "#A1AFB1"
+GLOBAL_LIST_INIT(freon_color_matrix, list("#2E5E69", "#60A2A8", "#A1AFB1", rgb(0,0,0)))
 
 /obj/proc/make_frozen_visual()
 	// Used to make the frozen item visuals for Freon.
 	if(resistance_flags & FREEZE_PROOF)
 		return
 	if(!(flags_2 & FROZEN_2))
 		name = "frozen [name]"
-		add_atom_colour(list(FROZEN_RED_COLOR, FROZEN_GREEN_COLOR, FROZEN_BLUE_COLOR, rgb(0,0,0)), TEMPORARY_COLOUR_PRIORITY)
+		add_atom_colour(GLOB.freon_color_matrix, TEMPORARY_COLOUR_PRIORITY)
 		alpha -= 25
 		flags_2 |= FROZEN_2
 
 //Assumes already frozed
 /obj/proc/make_unfrozen()
 	if(flags_2 & FROZEN_2)
 		name = replacetext(name, "frozen ", "")
-		remove_atom_colour(TEMPORARY_COLOUR_PRIORITY, list(FROZEN_RED_COLOR, FROZEN_GREEN_COLOR, FROZEN_BLUE_COLOR, rgb(0,0,0)))
+		remove_atom_colour(TEMPORARY_COLOUR_PRIORITY, GLOB.freon_color_matrix)
 		alpha += 25
 		flags_2 &= ~FROZEN_2
 
-#undef FROZEN_RED_COLOR
-#undef FROZEN_GREEN_COLOR
-#undef FROZEN_BLUE_COLOR
-

```

Comment:
Perhaps we should add an undef here?

Reply:
undefs are actually more work for the preprocessor. If they aren't used anywhere it's generally not an issue

Reply:
there's nothing being defined any more regardless